<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteItemDialog</class>
 <widget class="QDialog" name="deleteItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Delete Item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="confirmation">
     <property name="text">
      <string>Are you sure you want to delete this item?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
